src/cloud-cli/meltano/cloud/README.md

```diff
@@ -0,0 +1,85 @@
+# Meltano Cloud CLI
+
+`meltano-cloud` is a standalone CLI tool for interacting with the Meltano Cloud API.
+
+## Configuration
+
+`meltano-cloud` can be configured via
+`meltano-cloud config set <setting name> <setting value>`.
+
+### Configuration File
+
+Meltano Cloud configuration settings and credentials are stored in a JSON file.
+`meltano-cloud` searches for this configuration file using [platformdirs](https://github.com/platformdirs/platformdirs).
+
+If a configuration file is found but the configuration it contains
+is invalid, `meltano-cloud` will raise an error rather than continuing its search.
+
+### Settings
+
+The following settings are configurable:
+
+#### `auth_callback_port`
+
+As part of authentication, `meltano-cloud` runs a lightweight, short-lived HTTP
+server on `localhost` to handle authentication callbacks. By default, this
+server runs on port 9999 but can be configured to another port using this setting.
+
+- Command line flag: `--auth-callback-port`
+- Environment variable: `MELTANO_CLOUD_AUTH_CALLBACK_PORT`
+
+#### `base_url`
+
+This is the base URL used for API calls to Meltano Cloud, `https://api.meltano.cloud`
+by default.
+
+- Command line flag: `--base-url`
+- Environment variable: `MELTANO_CLOUD_BASE_URL`
```

Comment:
Should this have the `internal.` prefix?

```suggestion
This is the base URL used for API calls to Meltano Cloud, `https://internal.api.meltano.cloud`
```

Reply:
Yep, will update.